### ISSUE TITLE
Sculpting tool bar missing in Godot 4.2.stable (check Project Settings->Plugins and enable)

### ISSUE THREAD

**Issue by kawht** [TokisanGames/Terrain3D#289]
### Terrain3D version

v0.9.0-beta

### System information

Godot v4.1.3.stable - Windows 10/64 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 4070 Desktop GPU (NVIDIA; 31.0.15.4633)

### Is the issue reproducable in the demo?

Yes

### Issue description

After reloading projects, and restarting Godot a bunch, I'm able to see the toolbar in the demo.
But not in any other project! And even still, it required a lot of fuss. More than just reloading twice.
I tried making terrain3d the root node, changing file structures, re-installing godot, terrain3d, and projects, etc. 

I've read other people are having the same issue, but their fixes aren't working for me.

Let me know what I can do to help, I can't get logs working.

**Comment by kawht** [comment]
![cap1](https://github.com/TokisanGames/Terrain3D/assets/13322922/4bb57adf-ec9d-4301-9ab9-6c57e66329f3)

**Comment by TokisanGames** [comment]
4.3 is most definitely not supported, nor stable itself. But your report says 4.1.3, and your picture says 4.2.

So, in 4.1.3 or 4.2.1, you were supposed to restart twice, as described in the instructions. And after doing so, you say it now works in the demo. Perhaps didn't install it properly in your own project. Did you watch the tutorial video linked in the Readme on the front page? 

Look in the documentation for running the engine with the console open which you should always do. Copy and paste everything in the console. No one can help you, not even yourself, if you don't read or share the error messages in your console. They most likely say the exact reason why it doesn't work.

**Comment by TokisanGames** [comment]
Also make sure the plugin is enabled in Project Settings/Plugins, per the instructions.

**Comment by kawht** [comment]
I appreciate your speedy response. I apologize for using an incorrect version in the title, I will fix. My issues only pertain to Godot 4.2.stable, I don't use 4.3 ever.
I already watched the whole video twice, read the docs, did everything exactly as prescribed and was still having issues. It was late at night, and I believe I missed the part about enabling the plugin. That, was my problem.

Once I enabled the plugin in Project Settings/Plugins, it finally worked. Very sorry for the waste of time :(
